### PR TITLE
[all lang] add a note for router name conventions

### PR DESCRIPTION
**Pull request by cake17**
I wasn't sure of the routes conventions for long names and found the info [here](http://sandbox3.dereuromark.de/pages/best-practices).
Please feel free to change the sentence if it's not correct in english.

### CODE REVIEW

File: en/intro/conventions.rst [hunk 1]
```diff
@@ -42,6 +42,16 @@ the convention is that your URLs are lowercase and dashed using the
 ``DashedRoute`` class, therefore ``/red-apples/go-pick`` is the correct form
 to access the ``RedApplesController::goPick()`` action.
 
+When you create links using ``this->Html->link()``, you can use the following
+conventions for the url array::
+
+    $this->Html->link('link-title', [
+        'prefix' => 'my-prefix // unmodified
+        'plugin' => 'MyPlugin, // camelCased
+        'controller' => 'ControllerName', // camelCased
```
Comment (HavokInspiration) [inline en/intro/conventions.rst]: I'd rather write it ``CamelCased`` (with a capital C) to better reflect the "format".

Comment (cake17) [inline en/intro/conventions.rst]: that's right, I'll change that

File: fr/intro/conventions.rst [hunk 2]
```diff
@@ -43,6 +43,16 @@ la convention est que vos URLs soient en minuscules avec des tirets en utilisant
 la classe ``DashedRoute``, donc ``/red-apples/go-pick`` est la bonne forme
 pour accéder à l'action ``RedApplesController::goPick()``.
 
+Quand vous créez des liens en utilisant ``this->Html->link()``, vous pouvez
+utiliser les conventions suivantes pour le tableau d'url::
+
+    $this->Html->link('link-title', [
+        'prefix' => 'my-prefix // unmodified
```
Comment (markstory) [inline fr/intro/conventions.rst]: I don't think namespaces can have `-` in them. This sounds like another key that DashedRoute should be inflecting.

Comment (cake17) [inline fr/intro/conventions.rst]: So the correct convention for prefix would be `myPrefix` or  `MyPrefix` ?

Comment (markstory) [inline fr/intro/conventions.rst]: I think MyPrefix makes sense as that's the name of the namespace. Looking at InflectedRoute and the Default route class, neither of those inflect prefix - which is also odd.

**Comment by cake17** [comment]
Prefix is now in camelcase

**Pull request merged**